tiledb/sm/group/group.h

```diff
@@ -117,11 +117,24 @@ class Group {
   void clear();
 
   /**
-   * Deletes data from and closes a group opened in MODIFY_EXCLUSIVE mode.
+   * Performs deletion of data and closes the local group with the given URI.
    *
-   * Note: if recursive == false, data added to the group will be left as-is.
+   * @param uri The URI of the group to be deleted.
+   * @param vfs The virtual filesystem on which the local group sits.
+   * @param tp The compute thread pool.
    *
-   * @param uri The address of the group to be deleted.
+   */
+  static void delete_group(
+      const URI& uri, VFS* vfs, tiledb::common::ThreadPool* tp);
```

Comment:
This function is only called once, from a member function of the same name. It would be better to inline the function entirely, or perhaps convert it into a member function (_i.e._ not `static`) called something other than `delete_group`.